docs/development/doc_guide.rst

```diff
@@ -1,76 +1,633 @@
+
+in order to build the documentation without executing the
+:ref:`example notebooks <example_notebooks>`. This command will pass
+even if there are warnings.  The `tox`_ command may also include the
+``--parallel`` flag to allow a parallel build.
+
+If you have `make <https://www.gnu.org/software/make/>`_ installed,
+then you can build the documentation by entering the ``docs/`` directory
+and running
+
+.. code-block:: bash
+
+   make html
+
+Including the ``-j n`` flag in the ``make`` command will enable a
+parallel build, where ``n`` is replaced with the number of processes.
+
+Documentation tools
+===================
+
+ReStructuredText
+----------------
+
+PlasmaPy's documentation is written using the `reStructuredText (reST)
 <https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html>`_
-of Sphinx's website.
+markup language. reST is human readable when viewed within a
+source code file or when printed out using `help`. reST also contains
+markup that allows the text to be transformed into `PlasmaPy's online
+documentation`_. reST files end in ``.rst``. Documentation contained
+within ``.py`` files is written in reST.
 
-Using sphinx within the project
--------------------------------
-To build docs locally, either:
+ReStructuredText Examples
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* use `Tox <https://tox.readthedocs.io/en/latest/>`_ with ``tox -e build_docs`` from within the main PlasmaPy repository directory, or
-* enter the ``docs`` directory and run ``make html``.
+Here we show some examples of reST that are commonly used in PlasmaPy.
 
-Afterwards, open ``docs/_build/html/index.html`` with your browser of choice.
+This is an example of including headings for the document title,
+sections, subsections, and so on. The lines surrounding each heading
+are the same length as that heading.
 
-Do try to solve warnings in documentation when writing your code. To enforce this,
-The ``build_docs`` environment is set to fail on encountering any warnings via
-the ``-W`` flag to ``sphinx-build``
+.. code-block:: rst
 
-.. note::
-   The ``tox -e build_docs_no_examples`` command will build the documentation without
-   executing the :ref:`example notebooks <example_notebooks>`. It will also
-   pass with warnings.
+   ==============
+   Document title
+   ==============
+
+   Heading 1
+   =========
+
+   Heading 2
+   ---------
+
+   Heading 3
+   ~~~~~~~~~
+
+We can link to code objects by enclosing them in back ticks.
+
+.. code-block:: rst
+
+  Here is a reference to `plasmapy.particles` that will write out the
+  full namespace when `sphinx` generates the documentation and generates
+  the link. Only the word "Particle" will show up if we prepend a
+  tilde like in `~plasmapy.particles.particle_class.Particle`.
+
+This linking will work for `python` commands as well as commonly used
+packages like `numpy`, `astropy`, `scipy`, and `pandas`. Intersphinx_
+mappings are defined in the ``intersphinx_mapping`` variable in
+`docs/conf.py`_.
```

Comment:
```suggestion
This linking will work for `python` commands as well as commonly used
packages like `numpy`, `astropy`, `scipy`, and `pandas`.   This cross-package
linking is made possible with the `sphinx.ext.intersphinx` extension and
the sources used for this cross-linking is defined in the
``intersphinx_mapping`` variable in `docs/conf.py`_.  If a cross-link is not
working as expected this is usually due to (1) a typo, (2) the package is
not defined in the ``intersphinx_mapping`` configuration value, or (3)
the referenced source package does not properly or fully index their
own code (this is common in open-source packages).
```

Reply:
@namurphy Why did you mark this as resolved?  What is your take on this?

Reply:
Huh...I have no memory of marking this or other comments as resolved.  It could have been that I made changes to those lines of code which marked these comments as "outdated"...or possibly that I forgot how to use GitHub.  Whatever the case, I liked this suggestion.